pom.xml

```diff
@@ -479,6 +472,7 @@
 		      <version>1.5</version>
 		       <configuration>
 			      <executable>gpg</executable>
+			      <skip>true</skip>
```

Comment:
I left it for now, otherwise it fails building in jenkins. I will remove it before merge. Once its jar is placed in our artifactory, I don't need to build it in jenkins anymore.